docs/resources/prefix.md

```diff
@@ -39,9 +39,12 @@ resource "netbox_prefix" "my_prefix" {
 - `custom_fields` (Map of String)
 - `description` (String)
 - `is_pool` (Boolean)
+- `location_id` (Number) Conflicts with `site_id`, `site_group_id` and `region_id`.
 - `mark_utilized` (Boolean)
+- `region_id` (Number) Conflicts with `location_id`, `site_id` and `site_group_id`.
 - `role_id` (Number)
-- `site_id` (Number)
+- `site_group_id` (Number) Conflicts with `location_id`, `site_id` and `region_id`.
+- `site_id` (Number) Conflicts with `location_id`, `site_group_id` and `region_id`.
```

Comment:
> > I noticed prefix are getting `region_id`, `site_id` and `site_group_id`. Probably time to add as well for `ip_address` and `ip_range`?
> 
> These objects do not have the attributes you mentioned.

@fbreckle this one has.

Reply:
I do not understand WHAT you want to be added WHERE.

Please elaborate.